docs/articles/Core/Serialization/index.md

```diff
@@ -8,6 +8,6 @@ The platform uses two different ways of serializing objects - JSON and our custo
 
 The three sections below explain the entry format, how it is created from objects or classes on the server and how to use it on the client.
 
-* [Entry Format](xref:EntryFormat)
-* [Entry Convert](xref:EntryConvert)
-* [Entry To Model](xref:EntryToModel)
+* [Entry Format](EntryFormat)
```

Comment:
those links don't work. Links always have to be to a concrete file. There should be markdown files for that